packages/typegpu/tests/resolve.test.ts

```diff
@@ -176,6 +176,108 @@ describe('tgpu resolve', () => {
     );
   });
 
+  it('should resolve an unstruct to its corresponding struct', () => {
+    const vertexInfo = d.unstruct({
```

Comment:
Let's stick to PascalCase for structs.

```suggestion
    const VertexInfo = d.unstruct({
```